scripts/run_locust.sh

```diff
@@ -1,15 +1,17 @@
 #!/bin/bash
 
-poetry run locust -f tests/locust/test_no_errors.py --html tests/locust/no_errors.html --headless --host $HOST_URL || true;
+poetry run locust -f tests/locust/tests/test_no_errors.py --html tests/locust/results/no_errors.html --headless --host $HOST_URL || true;
```

Comment:
I believe this will error if the results folder doesn't exist, can we get a mkdir before to ensure its created, plus can we add this tests/locust/results into the .gitignore file so we don't accidentally commit results